heltour/tournament/models.py

```diff
@@ -1423,6 +1424,9 @@ class PlayerPairing(_BaseModel):
     game_link = models.URLField(max_length=1024, blank=True, validators=[game_link_validator])
     scheduled_time = models.DateTimeField(blank=True, null=True)
     colors_reversed = models.BooleanField(default=False)
+    
+    #We do not want to mark players as unresponsive if their opponents got assigned after round start
+    last_time_player_changed = models.DateTimeField(blank=True, null=True)
```

Comment:
To be consistent, we should probably use a name that starts with `date_`, like the `date_modified` or `date_created` attributes.  Maybe something like:
`date_players_modified`